app/Http/Controllers/ProductController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Product;
+use Illuminate\Http\Request;
+use App\Traits\Common;
+
+class ProductController extends Controller
+{
+    use Common;
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        // $products = Product::get();
+        //dd($products);
+        //1st solution
+        //$products = Product::orderByDesc('created_at')->skip(0)->take(3)->get();
+        
+        //2nd solution as the 1st one
+        //$products = Product::latest()->take(3)->get();
+
+        //3rd soln.
+        $latestProductsDesc = Product::latest()->take(3)->get();
+        $products = $latestProductsDesc->sortBy('created_at');
```

Comment:
this code is redundant look at the internal implementation of latest function
```php
public function latest($column = 'created_at')
    {
        return $this->orderBy($column, 'desc');
    }
```

the results are sorted